### PR TITLE
Update shadow.py

### PR DESCRIPTION
**Pull request by Relativist1**
Fixes #<398>

### CODE REVIEW

File: src/einsteinpy/rays/shadow.py [hunk 1]
```diff
@@ -88,6 +114,10 @@ def _intensity_from_event_horizon(self):
         return k1
 
     def smoothen(self, points=500):
+        '''
+        Returns the interpolated values for the intensities for smoothening of the plot
```
Comment (ritzvik) [inline src/einsteinpy/rays/shadow.py]: It does not return anything BTW.
```suggestion
        Sets the interpolated values for the intensities for smoothening of the plot
```

File: src/einsteinpy/rays/shadow.py [hunk 2]
```diff
@@ -35,15 +36,26 @@ def __init__(self, mass, n_rays, fov, limit=0.001):
         self.k0 = self._intensity()
         self.k1 = self._intensity_from_event_horizon()
         self.intensity = self.k1 + self.k0
-
+        def _compute_B(self):
+        '''
+        Returns an array of impact parameters
+        '''
+        return np.linspace(self.b_crit.value, self.distance, self.n_rays)
```
Comment (ritzvik) [inline src/einsteinpy/rays/shadow.py]: was this function `_compute_B` required? I guess this is already handled by this.

https://github.com/einsteinpy/einsteinpy/blob/657330b0fda1f80c429f7eb025501d23fcdf848a/src/einsteinpy/rays/shadow.py#L25

Anyway, remove this change if you feel this was redundant.

**Comment by ritzvik** [comment]
Also to pass quality checks, see [this](https://docs.einsteinpy.org/en/latest/dev_guide.html#code-linting)

**Comment by shreyasbapat** [comment]
I am taking over. Well, because we are lacking time!

**Comment by shreyasbapat** [comment]
Okay, no. There's a lot of work to do. Postponing this!

**Comment by shreyasbapat** [comment]
@ritzvik If you can land this PR in tomorrow, that would be great.

**Comment by ritzvik** [comment]
First thing in the morning. Netflix right now!

Regards,
Ritwik

File: src/einsteinpy/geodesic.py [hunk 3]
```diff
@@ -27,12 +27,6 @@ def __init__(
             Defaults to ``1e-3``.
         time : ~astropy.units.s, optional
             Time of start, Defaults to 0 seconds.
-        a : ~astropy.units.m, optional
-            Spin factor of massive body. Should be less than half of schwarzschild radius.
-        q : ~astropy.units.C, optional
-            Charge on the massive body
-        parent : ~einsteinpy.bodies.Body, optional
-            The parent object of the body.
```
Comment (shreyasbapat) [inline src/einsteinpy/geodesic.py]: Oh my god. How did it get through :(

Comment (ritzvik) [reply src/einsteinpy/geodesic.py]: :(

**Comment by ritzvik** [comment]
@Relativist1 sorry that I had to take over your PR, given the urgency of the release.
BTW, it's great to have you as a community member!

**Pull request merged**